Pipfile

```diff
@@ -26,7 +25,9 @@ jsonschema = "~=3.2"
 "kids.cache" = "==0.0.7"
 netaddr = "~=0.7"
 ordered-set = "~=3.1"
-psycopg2 = "~=2.8"
+# psycopg2 doesn't use semver:
```

Comment:
not the real reason to lock it to `2.8.*`, add note about incompatibility with Django 2.2